tests/periph_uart/tests/02__periph_uart_mode.robot

```diff
@@ -0,0 +1,68 @@
+*** Settings ***
+Suite Setup         Run Keywords    Reset DUT and PHILIP
+...                                 DUT Must Have Periph UART Application
+Test Setup          Run Keywords    Reset DUT and PHILIP
+...                                 DUT Must Have Periph UART Application
+
+Resource            periph_uart.keywords.txt
+Resource            api_shell.keywords.txt
+
+Variables           test_vars.py
+
+Force Tags          periph  uart
+
+*** Test Cases ***
+Even Parity 8 Bits
+    ${status}   ${value}=       Run Keyword And Ignore Error   API Call Should Succeed   Uart Mode   data_bits=8   parity="N"   stop_bits=1
```

Comment:
looks like these two lines are used in all tests here and should be wrapped into a "speaking" keyword?
Something like `DUT Should Have UART Mode` maybe (@MrKevinWeiss what do you think?)

Reply:
Maybe `UART mode should exist` or `DUT UART mode should exist`

Reply:
I've used "DUT UART mode should exist"